common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidPlatformUtil.java

```diff
@@ -213,6 +218,23 @@ public String getPackageNameFromUid(int uid) {
         return mContext.getPackageManager().getNameForUid(uid);
     }
 
+    @Override
+    public List<Map.Entry<String, String>> setPlatformSpecificExtraQueryParameters(@Nullable List<Map.Entry<String, String>> originalList) {
```

Comment:
Generally the purpose of a `setter` is to set or modify a value within `this` class i.e. a setter that lives on class `AndroidPlatformUtil` may modify a member variable of that class. We probably need a better name for this method, Maybe call it `updateAndGetQueryParamsWithPlatformSpecificQp`

Reply:
Maybe updateWithAndGetPlatformSpecificExtraQueryParameters? 4e76e7f2bd318e2245fb36dd47b7494e23c4a845